lib/v2/datasource/remote/model/transaction_model.dart

```diff
@@ -0,0 +1,47 @@
+class TransactionModel {
+  final String from;
+  final String to;
+  final String quantity;
+  final String memo;
+  final String timestamp;
+  final String transactionId;
+
+  String get symbol {
+    return quantity.split(" ")[1];
+  }
+
+  TransactionModel(this.from, this.to, this.quantity, this.memo, this.timestamp, this.transactionId);
```

Comment:
for more of one parameter, I recommned use named paramaters.
`({this.from, this.to, this.quantity, this.memo, this.timestamp, this.transactionId})`

Reply:
Oh for sure, did not notice this didn't have named params as I just copied the code out of the old models file. Changing.